From daniel.ritz@gmx.ch Thu Sep 29 05:07:08 2005
From: Daniel Ritz <daniel.ritz@gmx.ch>
To: Greg KH <gregkh@suse.de>, Vojtech Pavlik <vojtech@suse.cz>
Subject: [PATCH] usb/input/touchkit: add more device IDs
Date: Wed, 28 Sep 2005 21:30:12 +0200
Message-Id: <200509282130.12677.daniel.ritz@gmx.ch>

[PATCH] usb/input/touchkit: add more device IDs

add two more device IDs from eGalax' programming guide.
thanks to Jonathan Hopper <jrhopper () gmail ! com>  for pointing out.

Signed-off-by: Daniel Ritz <daniel.ritz@gmx.ch>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

 drivers/usb/input/touchkitusb.c |    2 ++
 1 file changed, 2 insertions(+)

---
--- gregkh-2.6.orig/drivers/usb/input/touchkitusb.c
+++ gregkh-2.6/drivers/usb/input/touchkitusb.c
@@ -75,7 +75,9 @@ struct touchkit_usb {
 
 static struct usb_device_id touchkit_devices[] = {
 	{USB_DEVICE(0x3823, 0x0001)},
+	{USB_DEVICE(0x0123, 0x0001)},
 	{USB_DEVICE(0x0eef, 0x0001)},
+	{USB_DEVICE(0x0eef, 0x0002)},
 	{}
 };
 
